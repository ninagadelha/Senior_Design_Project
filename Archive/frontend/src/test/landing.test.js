import renderer from "react-test-renderer";
import Home from "../routes/landing";

jest.mock("../styles/landing.css");
jest.mock("../index.css");
jest.mock("../components/landingNavbar");

const renderTree = (tree) => renderer.create(tree);
describe("<Home>", () => {
  it("should render component", () => {
    expect(renderTree(<Home />).toJSON()).toMatchSnapshot();
  });
});
