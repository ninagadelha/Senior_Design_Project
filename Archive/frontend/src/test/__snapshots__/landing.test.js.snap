// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home> should render component 1`] = `
<div
  className="main-container"
>
  <h1
    className="home-title"
  >
    MySTEMGrowth Profile
  </h1>
  <p
    className="home-body-text"
  >
    When youâ€™re in the thick of your studies, sometimes it can be challenging to appreciate where you are in your STEM journey. MySTEMGrowth Profile will allow you to step back and see where you are along your path. Our surveys explore your personal beliefs about your capabilities and how those might help your community. Take the surveys, explore your results, and celebrate your unique path in the world of science, technology, engineering, and mathematics.
  </p>
</div>
`;
